tools/iframe-test-resizer.html

```diff
@@ -0,0 +1,36 @@
+<!-- this file can be loaded directly from a browser and used to test embedding groups and projects within iframes
+see https://github.com/DemocracyLab/CivicTechExchange/issues/978 
+change groups/inframe to projects/inframe below to test embedding a project
+This if for testing with iframe resizer
+ -->
+<!DOCTYPE html>
+<html lang="en" style="height: 100%">
+  <head>
+    <meta charset="utf-8" />
```

Comment:
Missing the viewport meta tag here - makes testing go wrong without it.

Reply:
Done